content/cybersecurity/concepts/malware/malware.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Malware'
+Description: 'Malware is malicious software that is designed to invade, damage and disable computer systems, networks and devices.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Malware** is malicious software that is intentionally designed to infiltrate, harm, or disable computer systems, networks, and devices. Depending on the malware's objective, the type of malware can vary, from theft to financial gain to sabotaging the ability to access the system.
+
+## Types of Malware
+
+There are different types of malware for different objectives, some of which are:
+
+- [Virus](https://www.codecademy.com/resources/docs/cybersecurity/virus) - attach themselves to other programs and replicate once installed.
+- Spyware - monitors user activities and reports back to the author.
+- Adware - designed to show advertisements to trick users into installing malicious software.
+- [Ransomware](https://www.codecademy.com/resources/docs/cybersecurity/ransomware) - designed to lock and encrypt files to force the user into paying a ransom to regain access.
+- Trojans - most dangerous of malware types. It can be used to steal financial information or install other types of malware.
+- Worms - similar to viruses, they are self-replicating but have the ability to spread across systems on their own.
+- Keyloggers - malware that records the user's keystrokes, records the data, and reports back to the author.
+- Rootkits - provides "root" access to the author and is designed to stay hidden from the user, other software, and the operating system itself.
```

Comment:
```suggestion
- A [virus](https://www.codecademy.com/resources/docs/cybersecurity/virus) attached itself to other programs and replicates once installed.
- Spyware monitors user activities and reports back to the author.
- Adware is designed to show advertisements to trick users into installing malicious software.
- [Ransomware](https://www.codecademy.com/resources/docs/cybersecurity/ransomware) is designed to lock and encrypt files to force the user into paying a ransom to regain access.
- Trojans are the most dangerous type of malware. It can be used to steal financial information or install other types of malware.
- Worms are similar to viruses. They are self-replicating but have the ability to spread across systems on their own.
- Keyloggers are malware that records the user's keystrokes, records the data, and reports back to the author.
- Rootkits provide "root" access to the author and is designed to stay hidden from the user, other software, and the operating system itself.
```